src/lib/arrays.js

```diff
@@ -0,0 +1,118 @@
+const getNthElement = (index, array) => {
+  if (index < array.length){
+    return array[index];
+  }
+  return array[index % array.length];
+};
+
+const arrayToCSVString = array => {
+  return array.join(',');
+};
+
+const csvStringToArray = string => {
+  return string.split(',');
+};
+
+const addToArray = (element, array) => {
+  array.push(element);
+};
+
+const addToArray2 = (element, array) => {
+  const arrayedTwice = [...array];
+  arrayedTwice.push(element);
+  return arrayedTwice;
+
+  // const contain = [];
+  // contain.push(element);
+  // const newArrayx = array.concat(contain);
+  // return newArrayx;
+};
+
+const removeNthElement = (index, array) => {
+  array.splice(index, 1);
+};
+
+const numbersToStrings = numbers => {
+  return numbers.toString().split(',');
+};
+
+const uppercaseWordsInArray = strings => {
+  const convertedNames = [];
+  for (let i = 0; i < strings.length; i + 1) {
+    convertedNames[i] = strings[i].toUpperCase();
+  }
+  return convertedNames;
+};
+
+const reverseWordsInArray = strings => {
+  return strings.map(string => {
+    return string
+      .split('')
+      .reverse()
+      .join('');
+  });
+};
+
+const onlyEven = numbers => {
+  const hibby = numbers.filter(evenNumber => evenNumber % 2 === 0);
+  return hibby;
+};
+
+// ^^ The above are two examples of returning a function nested in another function.
+//Either with two return statements, or with one, having the nested function
+//assigned to an additional variable.
+
+const removeNthElement2 = (index, array) => {
+  return array.filter((_, indexRem) => {
+    return indexRem !== index;
+  });
+};
+
+// const elementsStartingWithAVowel = strings => {
+//   return strings.filter(str => /^[aeiou]/i.test(str));
+// };
```

Comment:
Theres a few more instances of code being commented out like this in the rest of the file, you could remove all of these and would make things a bit clearer